apps/etterlatte-behandling/src/test/kotlin/sak/SakDaoTest.kt

```diff
@@ -63,40 +63,48 @@ internal class SakDaoTest(val dataSource: DataSource) {
         behandlingRepo.opprettBehandling(opprettBehandling)
 
         // setter opprettet dato fra behandling
-        dataSource.connection.prepareStatement(
-            """
-            UPDATE sak as s
-            SET opprettet = (select behandling_opprettet from behandling as b where b.sak_id = s.id);
-            """.trimIndent(),
-        ).executeUpdate()
-
-        val saker =
-            dataSource.connection.prepareStatement(
+        dataSource.connection.use {
+            it.prepareStatement(
                 """
-                select * from sak;    
+                UPDATE sak as s
+                SET opprettet = (select behandling_opprettet from behandling as b where b.sak_id = s.id);
                 """.trimIndent(),
-            ).executeQuery().toList {
-                Pair(getString("fnr"), getTidspunktOrNull("opprettet"))
+            ).executeUpdate()
+        }
+
+        val saker =
+            dataSource.connection.use {
+                it.prepareStatement(
+                    """
+                    select * from sak;    
+                    """.trimIndent(),
+                ).executeQuery().toList {
+                    Pair(getString("fnr"), getTidspunktOrNull("opprettet"))
+                }
             }
         val sakmedTidligereDato = saker.find { it.first === fnrMedBehandling }
 
         sakmedTidligereDato?.second?.toLocalDate()?.shouldBeToday()
 
-        dataSource.connection.prepareStatement(
-            """
-            UPDATE sak
-            SET opprettet = 'yesterday'::TIMESTAMP
-            WHERE sak.opprettet IS NULL;
-            """.trimIndent(),
-        ).executeUpdate()
-
-        val beggesakerMedDato =
-            dataSource.connection.prepareStatement(
+        dataSource.connection.use {
+            it.prepareStatement(
                 """
-                select * from sak;    
+                UPDATE sak
+                SET opprettet = 'yesterday'::TIMESTAMP
+                WHERE sak.opprettet IS NULL;
                 """.trimIndent(),
-            ).executeQuery().toList {
-                Pair(getString("fnr"), getTidspunktOrNull("opprettet"))
+            ).executeUpdate()
+        }
+
+        val beggesakerMedDato =
+            dataSource.connection.use {
+                it.prepareStatement(
+                    """
+                    select * from sak;    
```

Comment:
pet peeve men selv om det er kun for test synes jeg at kolonnene bør eksplisitt angis

Reply:
[ba35602](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4621/commits/ba35602871cfd0f6d5b2311dbb94ca8eb4587dc1)